ontology/aao.md

```diff
@@ -1,17 +1,17 @@
 ---
-layout: ontology_detail
-id: aao
-title: Amphibian gross anatomy
```

Comment:
layout, id and title should be first (in that order), activity_status last.. The rest can be done alphabetically like you propose.

Reply:
The sorting is done by the YAML Dumper class, I'm not sure if the sort can be customized. I'll look into it, but no promises. Even if this isn't possible, I still think the consistent ordering is more valuable than the priorization

Reply:
I use a lot of "head" when working with these files locally, and also it helps me to quickly navigate the YAML file. I wouldn't say it for a cosmetic preference. The activity status isn't important, so you could split the YAML and dump twice.

Reply:
^ excellent thinking

Reply:
Done in c748947